permissions/plugin-maven-artifact-choicelistprovider.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "maven-artifact-choicelistprovider"
+paths:
+- "org/jenkins-ci/plugins/maven-artifact-choicelistprovider"
```

Comment:
Okay. Pushed my changes to the jenkins-fork and now the groupId should match the groupId in this push request.